custom/download_qrda.php

```diff
@@ -150,10 +150,10 @@ function closeMe() {
         <th scope="col" class="multiDownload">
             <input type="checkbox" name="checkAll" id="checkAll"/>
             <div style="display:none" id=downloadAll>
-                <img class='downloadIcon' src='<?php echo $GLOBALS['webroot'];?>/images/downbtn.gif' onclick=downloadAllXML(); />
+                <img class="downloadIcon" src="<?php echo $GLOBALS['webroot'];?>/downbtn.gif" onclick="downloadAllXML();" />
```

Comment:
This one has been migrated, so should use `$GLOBALS['images_static_relative']`:
```
<img class="downloadIcon" src="<?php echo $GLOBALS['images_static_relative']; ?>/downbtn.gif" onclick="downloadAllXML();" />
```

Reply:
Also agree with dropping the onclick there since that function does not exist:
```
<img class="downloadIcon" src="<?php echo $GLOBALS['images_static_relative']; ?>/downbtn.gif" />
```